### ISSUE TITLE
mpv with opengl output hog single CPU core, video lags a lot.

### ISSUE THREAD

**Issue by slashbeast** [mpv-player/mpv#1736]
Hi,

Since few days I've noticed that if I use output opengl or opengl-hq the video lags and single CPU core usage reaches 100%.

GPU: The one bundled within i7-4702MQ.

Packages:
media-libs/mesa-10.5.2
media-video/mpv-0.8.3
x11-drivers/xf86-video-intel-2.99.917

**Comment by slashbeast** [comment]
Closing at its surly not an mpv issue. Mplayer2 has the same problem and QT4-based apps have some huge lag redrawing.

**Comment by slashbeast** [comment]
The issue was within CONFIG_EXPERT that enables some extra debuging features. It was on for the whole time just got worst over adding more symbols into kernel.

Disabling CONFIG_EXPERT was the solution.

(Updated for the sake of people who will ever google it)

**Comment by Ardakilic** [comment]
I am having the same issue.

I had been using these builds for the time being:

http://sva.wakku.to/~chris/mpv_builds/

Now, by following this guide, 

http://coalgirls.wakku.to/faq/playback/compiling-mpv-on-mac-os-x

(only without the --HEAD flag so I've used releases to compile and install)

I had installed new version with homebrew.

I also have the `vo=opengl-hq:icc-profile-auto` in my config file.

But almost always in couple of minutes, in fullscreen, the video lags, when I go out of fullscreen, it's okay again.

What can be the problem, any ideas ?

Thanks,
